### PR TITLE
feat(update-prs): push new commits

### PR DESCRIPTION
**Pull request by Dan7-7-7**
https://github.com/Skyscanner/turbolift/issues/153

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -260,23 +260,23 @@ redacted/redacted                                         OPEN    REVIEW_REQUIRE
...
+If the flag `--yes` is not passed with an `update-prs` command, a confirmation prompt will be presented.
+As always, use the `--repos` flag to specify an alternative repo file to the default `repos.txt`.
 
-##### Close PRs with the `--close` flag
+##### Examples
 
 ```turbolift update-prs --close [--yes]```
+```turbolift update-prs --push [--yes]```
+```turbolift update-prs --amend-description [--yes]```
+```turblift update-prs --amend-description --description prDescriptionFile1.md```
```
Comment (rnorth) [inline README.md:275]:
```suggestion
```turblift update-prs --amend-description [--description prDescriptionFile1.md]```
```
? As the --description flag is optional?

File: cmd/updateprs/updateprs.go [hunk 2]
```diff
@@ -109,7 +115,7 @@ func runClose(c *cobra.Command, _ []string) {
 	// Prompting for confirmation
 	if !yesFlag {
 		// TODO: add the number of PRs that it will actually close
-		if !p.AskConfirm(fmt.Sprintf("Close %s campaign PRs for all repos in %s?", dir.Name, repoFile)) {
+		if !p.AskConfirm(fmt.Sprintf("Close %s campaign PRs for all repos in %s", dir.Name, repoFile)) {
```
Comment (Dan7-7-7) [inline cmd/updateprs/updateprs.go:118]: Didn't like the double question mark here
image

Comment (Dan7-7-7) [inline cmd/updateprs/updateprs.go:118]: didn't like the double question mark here
image

**Review by rnorth — APPROVED** [review]
Looks good, nice work!

**Comment by YashrajDigheTide** [comment]
@Dan7-7-7 Does the above implementation supports the push mechanism for already raised mass PR's ?

image

I have the use case where I have raised mass PR's and I need to push new commits to those PR's.

**Comment by YashrajDigheTide** [comment]
@Dan7-7-7 I can see your code in `main` branch is working as expected, but seems like its not part of the latest release 4.1.1 and 4.1.0 as well.

**Pull request merged**